readthedocsext/theme/templates/includes/elements/promotions/pull-request-previews.html

```diff
@@ -0,0 +1,10 @@
+<div class="ui card">
+  <div class="content">
+    <a class="header" target="_blank" href="https://docs.readthedocs.io/page/pull-requests.html">Preview changes before deploying!</a>
+    <div class="description">
+      Easily catch documentation formatting and display issues with Pull Request Previews.
+      Stop worrying about having an up-to-date local build environment to build the docs locally,
+      and always have a sharable link with production-ready documentation before going live.
```

Comment:
For these messages, I'd aim for rather concise language and focus on individual user problems.

```suggestion
      Automatically build, preview, and share documentation with every pull request
      and easily catch issues before going live.
```